xonix-main-menu.js

```diff
@@ -0,0 +1,143 @@
+const figlet = require('figlet');
```

Comment:
Szerintem maradhat a stdin is. Valahogy le lehet tiltani a gombnyomásokat, amíg le nem fut a kezdőképernyő, aztán meg engedélyezni. Ha nincs rá semmi, akkor legrosszabb esetben egy if-el.